webpages/settings/data/example-manifest.js

```diff
@@ -0,0 +1,16 @@
+export default {
+  name: "",
+  description: "",
+  credits: [
+    {
+      name: "Credit",
```

Comment:
Can't we remove this or set it to blank (not "Credits")?

Reply:
Yes, it can be removed. I will remove it later today.